src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1387,18 +1386,22 @@ private static void validateViolationSection(String fileName, String sectionName
                 .isEqualTo("");
         }
         else {
+            final String subsectionTextContent = subSection.getTextContent()
+                    .replaceAll("\\n\\\\s+", "\n")
```

Comment:
Original regexp was `\n\\s+`. Now it is `\\n\\\\s+`, which has an extra set of `\`s.
`\\\\s+` will not match white spaces.

I am confused by this change.

Reply:
This wasn't intended. I probably did it fixing https://github.com/checkstyle/checkstyle/pull/13362#discussion_r1258232047.

Reverting.

Reply:
Orig: `"\n\\s+", "\n"`
New: `"\\n\\s+", "\n"`

Shouldn't the `\\n` be `\n`?

Reply:
Hoenstly, I don't know. I am moving it back to `\n` just so it's consistent with others. I see both `\n` and `\\n` being used around the repository.

Reply:
Yes, it should be `\n`

https://stackoverflow.com/a/59192811/9553927